web/src/locales/en.json

```
@@ -52,6 +52,10 @@
     "preliminary-data": "Preliminary data may change over time.",
     "summary": "Discover real-time electricity insights with the Electricity Maps app!"
   },
+  "new-feature-popover": {
+    "title": "Select a Date to Explore",
+    "content": "Use the date picker to view historical data on electricity sources, CO2 emissions, and prices."
+  },
```

Do you think we should add another layer in this if we will add more feature popovers in the future? 
Maybe something like this:

```suggestion
  "new-feature-popover": {
    "date-picker": {
      "title": "Select a Date to Explore,"
      "content": "Use the date picker to view historical data on electricity sources, CO2 emissions, and prices."
    },
  },
```